test-workflows/tasks/generators/clusterloader/load.yaml

```diff
@@ -0,0 +1,127 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: clusterloader2
+spec:
+  description: >-
+    clusterloader2 task to run various types of cl2 tests on a given cluster.
+  params:
+  - name: giturl
+    description: "git url to clone the package"
+    default: https://github.com/kubernetes/perf-tests.git
```

Comment:
Instead of this, I'd love to have a set of CL2 YAMLs written to github somewhere and then we can just point specifically to those. Alternatively, we can define those YAMLs in Kubernetes configmaps.

Reply:
This is not just for yamls, also to build cl2 binary that we want to run the test against.

Reply:
See the other comment about building a minimal go image. Check out https://github.com/google/ko

Reply:
Spoke offline with Ellis. This isn't viable because we need aws-cli for kubeconfig to work with authentication with EKS. IOW we need to build aws-cli in any image we want to go with.